src/Type/Enum/CommentsConnectionOrderbyEnum.php

```diff
@@ -15,67 +15,67 @@ public static function register_type() {
-						'description' => 'Order by e-mail of the comment author.',
+						'description' => __( 'Order by e-mail of the comment author.', 'wp-graphql' ),
 						'value'       => 'comment_author_email',
 					],
 					'COMMENT_AUTHOR_IP'    => [
-						'description' => 'Order by IP address of the comment author.',
+						'description' => __( 'Order by IP address of the comment author.', 'wp-graphql' ),
 						'value'       => 'comment_author_IP',
 					],
 					'COMMENT_AUTHOR_URL'   => [
-						'description' => 'Order by URL address of the comment author.',
+						'description' => __( 'Order by URL address of the comment author.', 'wp-graphql' ),
 						'value'       => 'comment_author_url',
 					],
 					'COMMENT_CONTENT'      => [
-						'description' => 'Order by the comment contents.',
+						'description' => __( 'Order by the comment contents.', 'wp-graphql' ),
 						'value'       => 'comment_content',
 					],
 					'COMMENT_DATE'         => [
-						'descriotion' => 'Order by date/time timestamp of the comment.',
+						'description' => __( 'Order by date/time timestamp of the comment.', 'wp-graphql' ),
 						'value'       => 'comment_date',
 					],
 					'COMMENT_DATE_GMT'     => [
-						'descriotion' => 'Order by GMT timezone date/time timestamp of the comment.',
+						'description' => __( 'Order by GMT timezone date/time timestamp of the comment.', 'wp-graphql' ),
 						'value'       => 'comment_date_gmt',
 					],
 					'COMMENT_ID'           => [
-						'description' => 'Order by the globally unique identifier for the comment object',
+						'description' => __( 'Order by the globally unique identifier for the comment object', 'wp-graphql' ),
 						'value'       => 'comment_ID',
 					],
 					'COMMENT_IN'           => [
-						'description' => 'Order by the array list of comment IDs listed in the where clause.',
+						'description' => __( 'Order by the array list of comment IDs listed in the where clause.', 'wp-graphql' ),
 						'value'       => 'comment__in',
 					],
 					'COMMENT_KARMA'        => [
-						'description' => 'Order by the comment karma score.',
+						'description' => __( 'Order by the comment karma score.', 'wp-graphql' ),
 						'value'       => 'comment_karma',
 					],
 					'COMMENT_PARENT'       => [
-						'description' => 'Order by the comment parent ID.',
+						'description' => __( 'Order by the comment parent ID.', 'wp-graphql' ),
 						'value'       => 'comment_parent',
 					],
 					'COMMENT_POST_ID'      => [
-						'description' => 'Order by the post object ID.',
+						'description' => __( 'Order by the post object ID.', 'wp-graphql' ),
 						'value'       => 'comment_post_ID',
 					],
 					'COMMENT_TYPE'         => [
-						'description' => 'Order by the The type of comment, such as \'comment\', \'pingback\', or \'trackback\'.',
+						'description' => __( 'Order by the The type of comment, such as \'comment\', \'pingback\', or \'trackback\'.', 'wp-graphql' ),
```

Comment:
Can you double check how descriptions like this make their way to GraphiQL. 

I feel like I have a memory of escaped quotes causing some issues with tooling like GraphiQL. . .but I also might be making things up in my head. Would be good to double check tho.

Reply:
I see this in graphiql browser. 

![Screen Shot 2021-03-22 at 12 15 15 PM](https://user-images.githubusercontent.com/749603/112030465-4d6e3700-8b08-11eb-8815-50451c3fc8b6.png)